components/ActivityCard/index.js

```diff
@@ -1,27 +1,20 @@
 import Image from "next/image";
 import styled from "styled-components";
+import Link from "next/link";
+import StyledImageComponent from "../StyledImage";
```

Comment:
Why do you rename it here? You can do so if you want but I wouldn't recommend it. This may lead to a bad overview when your codebase grows. I recommend to use the same name as the component name here.